plugin/pxr/cmake/macros/Public.cmake

```diff
@@ -190,7 +190,6 @@ function(pxr_setup_python)
     install(
         FILES
           ${CMAKE_CURRENT_BINARY_DIR}/__init__.py
-          ${CMAKE_CURRENT_BINARY_DIR}/__init__.pyc
```

Comment:
There is no need to install compiled bytecode for __init__  either for python2 or 3.

In python3 compiled byte-code  are stored in `__pycache__`, that means we have to have copy the compiled bytecode from `__pycache__` which is not really necessary, hence removing this all together.